python/src/scipp/plot/__init__.py

```diff
@@ -28,10 +29,18 @@
         # If we are in an IPython kernel, select widget backend
         if "IPKernelApp" in ipy.config and not is_doc_build:
             mpl.use('module://ipympl.backend_nbagg')
-            # Hide the figure header:
-            # see https://github.com/matplotlib/ipympl/issues/229
-            from ipympl.backend_nbagg import Canvas
-            Canvas.header_visible.default_value = False
+            try:
+                # Hide the figure header:
+                # see https://github.com/matplotlib/ipympl/issues/229
+                from ipympl.backend_nbagg import Canvas
+                Canvas.header_visible.default_value = False
+            except ImportError:
+                warnings.warn(
+                    "The ipympl backend, which is required for "
+                    "interactive plots in Jupyter, was not found. "
+                    "Falling back to a static backend. Visit "
+                    "https://github.com/matplotlib/ipympl for "
+                    "instructions on how to install ipympl.", ImportWarning)
```

Comment:
This is proposing a pip installation of ipympl, will this then cause matplotlib (or other upstream dependencies) to be installed via pip which we already have in conda?